internal/pkg/service/buffer/config/config_test.go

```diff
@@ -106,15 +106,15 @@ sink:
                 diskSync:
                     # Sync mode: "disabled", "cache" or "disk". Validation rules: required,oneof=disabled disk cache
                     mode: disk
-                    # Wait for sync to disk OS cache or to disk hardware, depending on the mode. Validation rules: excluded_if= Mode disabled
+                    # Wait for sync to disk OS cache or to disk hardware, depending on the mode.
                     wait: true
```

Comment:
Unrelated: Fixed in the config test, the rules have been updated in a prev PR, tests don't run in the CI yet.